src/components/Editor/index.tsx

```diff
@@ -1,8 +1,8 @@
 /* eslint max-lines: ["error", 350] */
-/* eslint max-lines-per-function: ["error", 170] */
+/* eslint max-lines-per-function: ["error", 200] */
+/* eslint max-statements: ["error", 30] */
```

Comment:
if you agree with the below suggestions, i believe we can remove those rule overrides
```suggestion
```